testdata/expected/src/github.com/sgtest/go-misc/github.com/sgtest/go-misc/multiple_mains/GoPackage.graph.json

```diff
@@ -60,7 +60,7 @@
     {
       "UnitType": "GoPackage",
       "Unit": "github.com/sgtest/go-misc/multiple_mains",
-      "Path": "main2.go/main/x",
+      "Path": "main2.go/main/x/fixme",
```

Comment:
`Path` is user-visible in the URL. Could we go with something less obviously a bug?